includes/drush/custom/custom.drush.inc

```diff
@@ -86,6 +95,36 @@ function drush_custom_features_revert_site() {
   }
 }
 
+/**
+ * Checks for non-used modules in certain path crossreferenced with a makefile.
+ */
+function drush_custom_check_unused_modules($path = 'sites', $make = '../resources/site.make') {
+  // If referenced make file does not exist, trow a warning.
+  if (!file_exists($make)) {
+    drush_log(dt('Make file @make does not exist. Showing all disabled modules in @path.', array('@make' => $make, '@path' => $path)), 'warning');
+  }
+  else {
+    $make_file = drupal_parse_info_file($make);
+  }
+
+  // Get all modules.
+  $modules = drupal_system_listing("/\\.module\$/", $path, 'name', 0);
```

Comment:
There is a way to detect if site is available or not and log it to terminal if not?

Reply:
I'll have a look at this to see if we can gracefully exit the command with a message.

Reply:
Maybe ?
```
drush status bootstrap | grep -q Successful
```

Reply:
I checked what happens without a functional site:
```
    [error] Command check-unused-modules needs a higher bootstrap level to run - you will need to invoke drush from a more functional Drupal environment to run this command.
    [error] The drush command 'check-unused-modules' could not be executed.
```

This seems like an appropriate error message no?